src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AdminController.cs

```diff
@@ -217,7 +217,7 @@ IOpenIdService openIdService
             }
 
             // If we got this far, something failed, redisplay form
-            return View("Edit",model);
+            return View("Create", model);
```

Comment:
@jersiovic FYI, the view name didn't match the action name, which causes view model type issues at runtime.